src/lib.rs

```diff
@@ -219,3 +219,6 @@ pub mod analysis {
     pub mod cross_validation;
     pub mod score;
 }
+
+/// Module for datasets.
+pub mod datasets;
```

Comment:
We should feature gate this. My thinking is that if we have a few datasets users will not want to download all of this data by default.

To do this:

- Add a new feature to [`Cargo.toml`](https://github.com/AtheMathmo/rusty-machine/blob/c7dcaaf0daf867b8f0d217f81a81c67d654b8b87/Cargo.toml#L14)
- In [`lib.rs` add a feature flag](https://github.com/AtheMathmo/rusty-machine/blob/3ab5a0513d4fc4aa7a47cd7c97c280cff66902f2/src/lib.rs#L210-L212)